### DISCUSSION TITLE
科学、工具与技术的诞生

### DISCUSSION THREAD

**Discussion by giscus** [zhuangbiaowei/zhuangbiaowei.github.io#20] [Announcements]
# 科学、工具与技术的诞生

这是一篇思考速记，并不能算是“成品”，因为一个庞大的思考轨迹，需要经常记录一些零星的收获，以便将来能够更好的串联、整理并正式发表。

https://zhuangbiaowei.github.io/2024/01/08/the-birth-of-tools-science-and-technology.html

**Comment by ZoomQuiet** [comment]
有一个问题, 内外并不是静态的...
随着内部已知的扩张, 第一个结果就是接触到更多未知的边界, 也就是:
+ 随着自己知道的越来越多
+ 导致知道自己不知道的越来越多

当然, 学习的意义, 就应该是人生的意义,
只是, 普通人从开始工作, 就已经丧失了探索自身好奇心的能力;
因为, 必须先将自己变成一个合格的社会劳动单元, 才能获得自己的生存位置;

所以, 普通人一生中真正主动学习的过程, 越来越压缩到幼儿园之前的那几年...
之后, 都被集体主义思想化了...

以及, 人类的好奇心和动物的好奇心, 有个本质区别在:
+ 人类满足好奇心的交付物和动物的根本不同
+ 动物是形成自己的经验就完了
+ 人类, 则倾向于: 
++ 可以向社会证明的作品
++ 可以反馈给社会的新故事/术语/概念/...
也就是我们, 我们学习的动力, 来自于更加倾向输出注入意义的好奇心:
比如, 通过我首先探索, 让大家知道有个什么好玩的事儿, 可以分几步, 几个级别玩到爆...
也就是将一个模糊的好奇心, 变成一系列有结构的好奇心了...

**Reply by zhuangbiaowei** [reply]
“普通人从开始工作, 就已经丧失了探索自身好奇心的能力;”  这个话其实我是不同意的，有些人在工作之前，可能就已经丧失了好奇心，有些人即使七老八十，依然有旺盛的好奇心。

人类的好奇心，本质上是“追问”的，不仅仅是发现新奇的现象，更希望能够问一个为什么。
